themes/frontierline/content-views/content-mini.php

```diff
@@ -2,17 +2,23 @@
 /**
  * Display a post with just the thumbnail, title, and date.
  */
+$post_classes = 'post-mini';
+if (get_theme_mod('frontierline_post_thumbnail') !== '1') {
+  $post_classes .= ' post-thumb-disabled';
```

Comment:
Question: Something like [`this`](https://codex.wordpress.org/Function_Reference/post_class#Add_Classes_By_Filters) may be more universal for all post classes in general.